examples/ASDKgram/Sample/PhotoFeedNodeController.m

```diff
@@ -192,7 +192,7 @@ - (void)tableView:(ASTableView *)tableView willBeginBatchFetchWithContext:(ASBat
 - (void)resetAllData
 {
   [_photoFeed clearFeed];
```

Comment:
@hannahmbanana Change `tableView:willBeginBatchFetchWithContext:` above to `tableNode:willBeginBatchFetchWithContext:`. Is that not showing a deprecation warning? Bummer.